app/src/main/java/com/tr4n/moviedb/ui/search/SearchFragment.kt

```diff
@@ -49,6 +49,7 @@ class SearchFragment : BaseFragment(R.layout.fragment_search) {
         })
         movieSearchAdapter.onItemSelected = { movieId ->
             activity?.supportFragmentManager?.beginTransaction()
+                ?.hide(this)
                 ?.replace(R.id.container, MovieDetailFragment.newInstance(movieId))
```

Comment:
không hide. Em thực hiện add chồng lên. Khi back nó sẽ quay trở lại màn Search 
```suggestion
                ?.add(R.id.container, MovieDetailFragment.newInstance(movieId))
           
```